### PR TITLE
TF: Add sigmoid activation function

### LINKED ISSUE #16810: Missing activation Function
I think the sigmoid / softmax activation function is missing here

https://github.com/huggingface/transformers/blob/31ec2cb2badfbdd4c1ac9c6c9b8a74e974984206/src/transformers/models/roberta/modeling_tf_roberta.py#L1299

### PR DESCRIPTION
**Pull request by gante**
# What does this PR do?

Fixes #16810 -- adds the sigmoid activation function to the TF activation functions. 

Also sorts the activation functions in the enum-like dict and in the tests, to quickly identify missing functions.

### CODE REVIEW

File: src/transformers/activations_tf.py [hunk 1]
```diff
@@ -113,16 +113,17 @@ def approximate_gelu_wrap(x):
...
     "gelu_new": gelu_new,
+    "glu": glu,
     "mish": mish,
-    "tanh": tf.keras.activations.tanh,
-    "gelu_fast": gelu_fast,
     "quick_gelu": quick_gelu,
-    "gelu_10": gelu_10,
-    "glu": glu,
+    "relu": tf.keras.activations.relu,
+    "sigmoid": tf.keras.activations.sigmoid,
```
Comment (gante) [inline src/transformers/activations_tf.py:123]: the only new line (in practice)

File: tests/utils/test_activations_tf.py [hunk 2]
```diff
@@ -42,17 +42,18 @@ def test_gelu_10(self):
...
-        get_tf_activation("gelu_fast")
         get_tf_activation("gelu_10")
+        get_tf_activation("gelu_fast")
+        get_tf_activation("gelu_new")
+        get_tf_activation("glu")
         get_tf_activation("mish")
         get_tf_activation("quick_gelu")
-        get_tf_activation("glu")
+        get_tf_activation("relu")
+        get_tf_activation("sigmoid")
```
Comment (gante) [inline tests/utils/test_activations_tf.py:53]: and its corresponding test :)

**Review by sgugger — APPROVED** [review]
Thanks for adding this and cleaning up!

**Review by Rocketknight1 — APPROVED** [review]
